test/src/databases/all/expr.spec.ts

```diff
@@ -340,6 +340,170 @@ describe.each(runtimes.runtimeList)('%s', (databaseName, runtime) => {
     });
   });
 
+  describe('sql expr functions', () => {
+    it('sql_string', async () => {
+      await expect(`
+      ##! experimental { sql_functions }
+      source: a is ${databaseName}.table('malloytest.aircraft_models') extend { where: aircraft_model_code ? '0270202' }
+
+      run: a -> {
+          group_by: string_1 is sql_string("UPPER(\${manufacturer})")
+        }
+      `).malloyResultMatches(expressionModel, {
+        string_1: 'AHRENS AIRCRAFT CORP.',
+      });
+    });
+
+    it('sql_number', async () => {
+      await expect(`
+      ##! experimental { sql_functions }
+      source: a is ${databaseName}.table('malloytest.aircraft_models') extend { where: aircraft_model_code ? '0270202' }
+
+      run: a -> {
+          group_by: seats
+          group_by: number_1 is sql_number("\${seats} * 2")
+        }
+  `).malloyResultMatches(expressionModel, {
+        seats: 29,
+        number_1: 58,
+      });
+    });
+
+    it('sql_boolean', async () => {
+      await expect(`
+      ##! experimental { sql_functions }
+      source: a is ${databaseName}.table('malloytest.aircraft_models') extend { where: aircraft_model_code ? '0270202' }
+
+      run: a -> {
+          group_by: boolean_1 is sql_boolean("\${seats} > 20")
+          group_by: boolean_2 is sql_boolean("\${engines} == 2")
```

Comment:
Looks like this should just be `sql_boolean("\${engines} = 2")` with one `=` rather than two.

Reply:
Ah, interesting.  I was using duckdb for local testing and this seems to pass 🤷   Clearly fails for postgres and bq so I'll be sure to run those locally too on future pulls to double check.